.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Logging**
+Add relevant SDK logging.  IMPORTANT NOTE:  Please make sure to NOT share AWS access credentials under any circumstance!  Please make sure they are not in the logs.
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**SDK version number**
+Include the SDK version you are running. If it is a specific commit on master branch, include that
+
+**Open source building**
+If it is a build issue, include 3rd party library version and steps to how you are building it
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```

Comment:
I would rather remove this section as most of the C producer users are not mobile clients. Instead, I would add something about the media pipeline

**Media pipeline:**
 - What is the media source (GStreamer, custom, etc). What is the upstream encoder/pipeline?
 - What is the media type, configuration
 - For Video - what's the codec information (Annex-B, AvCC, raw frames), CPD present, etc..
 - For Audio - what's the audio configuration (CPD generated, etc)..
 - Intermittent/continuous streaming?

Reply:
Yeah good point, Agreed, I'm going to update this.